content/en/docs/configuration/acme/dns01/google.md

```diff
@@ -35,6 +35,14 @@ $ gcloud projects add-iam-policy-binding $PROJECT_ID \
    --role roles/dns.admin
 ```
 
+> Note: The use of the `dns.admin` role in this example role is for convenience.
```

Comment:
```suggestion
> **Note**: The use of the `dns.admin` role in this example role is for convenience.
```

Just noticed we don't do this everywhere but it is nice to emphasis the note here.